File: Try_GameEngine5/src/com/example/try_gameengine/avg/AVGUtils.java
package com.example.try_gameengine.avg;

import java.util.concurrent.TimeUnit;

public class AVGUtils {

	public static void pause(long timeMillis) {
		try {
			TimeUnit.MILLISECONDS.sleep(timeMillis);
		} catch (InterruptedException e) {
			throw new RuntimeException("Interrupted in pause !");
		}
	}
}

File: Try_GameEngine5/src/com/example/try_gameengine/avg/AVGUtils.kt
package com.example.try_gameengine.avg

import java.util.concurrent.TimeUnit

object AVGUtils {
    fun pause(timeMillis: Long) {
        try {
            TimeUnit.MILLISECONDS.sleep(timeMillis)
        } catch (e: InterruptedException) {
            throw RuntimeException("Interrupted in pause !")
        }
    }
}
